src/planning/strategies/tests/map-slots-test.ts

```diff
@@ -154,13 +153,14 @@ ${recipeManifest}
     }
   });
 
-  it('prefers local slots if available', async () => {
+  // TODO(sjmiles): skipping because concept of local vs remote slots has changed
+  it.skip('prefers local slots if available', async () => {
```

Comment:
SlotContext is deprecated, because the Slot system no longer ever owns a node, that's all delegated to the UiBroker. The Slot system now is only concerned with virtual composition. 
The SlotUils::findAllSlotCandidates code relies on SlotComposer::getAvailableContexts to work out local vs remote slots; but as SlotContext is deprecated, getAvailableContexts no longer produces actionable data.

Reply:
> SlotContext is deprecated, because the Slot system no longer ever owns DOM nodes (or any other renderer object), instead delegating that responsibility to UiBroker. The Slot system now is only concerned with virtual composition. The SlotUils::findAllSlotCandidates code relies on SlotComposer::getAvailableContexts to work out local vs remote slots; but as SlotContext is deprecated, getAvailableContexts no longer produces actionable data.

I used to think of the `getAvailableContexts` as an implementation detail. Semantically, "local" are slots from within the current recipe, and "remote" - the ones already preexisting in the arc. I think this concept can (and should) still be maintained)